src/core/manager.c

```diff
@@ -1757,6 +1757,10 @@ static void manager_ready(Manager *m) {
                         UID_INVALID, GID_INVALID, 0444);
 
         m->honor_device_enumeration = true;
+
+        /* Re-read /proc/self/mountinfo when the first SIGCHLD event for mount unit is received. As, the
+         * mount states set by mount_enumerate() may be overwritten by the outdated deserialized states. */
```

Comment:
I think it's better to also mention that `mount_enumerate may failed to catch the change event`.

Reply:
What is "the change event"??

Reply:
I mean the change of `/proc/self/mountinfo`.
Emm... tought twice, I was wrong. The mount states set by `mount_enumerate()` should always be up-to-date, even if the libmount monitor is closed when systemd reexcutes.
The comment dosn't need any modify.